AutomatedTesting/Gem/PythonTests/Physics/tests/heightfield/AxisAlignedBoxShape_ConfigurationWorks.py

```diff
@@ -0,0 +1,101 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+#fmt: off
+class Tests():
+    create_test_entity                      = ("Entity created successfully",    "Failed to create Entity")
+    add_physx_heightfield_collider          = ("PhysX Heightfield Collider component added", "Failed to add PhysX Heightfield Collider component")
```

Comment:
Given what this test is validating, I don't think the PhysX Heightfield Collider is actually necessary.